streaming/receiver.c

```diff
@@ -640,6 +640,7 @@ static int rrdpush_receive(struct receiver_state *rpt)
                 rpt->host->hostname);
         }
     }
+    rpt->host->trigger_chart_obsoletion_check = now_realtime_sec();
```

Comment:
Don't you want to set this after the parser terminates?

Reply:
Hmm, sorry, not sure what you mean? I set it to 0 after the parser terminates... We basically want to trigger the check when the child connects. When the check happens, it will be set to 0 and will not retry, or if the child disconnects.